src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please, I observed that this linter was not disabled in [this given file](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js), kindly remove this and address the issue the linter in pointing at. The` import/no-extraneous-dependencies` rule is typically used to ensure that your project's dependencies are properly listed in your `package.json` file and not being imported as `extraneous dependencies`.

Reply:
Fixed.